### PR TITLE
Minor Improvements on Networkx/algorithms/community/quality.py

### PR DESCRIPTION
**Pull request by z3y50n**
Removed some unnecessary imports and fixed some minor docstring issues

### CODE REVIEW

File: networkx/algorithms/community/quality.py [hunk 1]
```diff
@@ -146,7 +148,7 @@ def performance(G, partition):
        Use `partition_quality` instead.
 
     The *performance* of a partition is the ratio of the number of
-    intra-community edges plus inter-community non-edges with the total
+    intra-community edges plus inter-community non-edges divided by the total
```
Comment (rossbar) [inline networkx/algorithms/community/quality.py:151]: The original wording had "the ratio of" which implies the division of the two quantities. I personally think your wording choice is better, but it should also include removing the "the ratio of" from the previous line.

File: networkx/algorithms/community/quality.py [hunk 2]
```diff
@@ -359,7 +361,7 @@ def partition_quality(G, partition):
     intra-community edges to the total number of edges in the graph.
 
     The *performance* of a partition is the ratio of the number of
-    intra-community edges plus inter-community non-edges with the total
+    intra-community edges plus inter-community non-edges divided by the total
```
Comment (rossbar) [inline networkx/algorithms/community/quality.py:364]: Ditto

**Review by rossbar — APPROVED** [review]
LGTM w/ one minor wording nit; otherwise straightforward improvements. Thanks @z3y50n !

File: networkx/algorithms/community/quality.py (hunk 1 above)
Comment (z3y50n) [reply networkx/algorithms/community/quality.py:151]: Yes you are right, I fixed it thanks

**Review by dschult — APPROVED** [review]

**Pull request merged**